### ISSUE TITLE
Show totals in labels page and parent objects

### ISSUE THREAD

**Issue by obarlev** [sysadminsmedia/homebox#328]
### What is the problem you are trying to solve with this feature?

Location page shows sum of all child items
For instance
![Image](https://github.com/user-attachments/assets/d7438248-6a09-437e-8946-5118f28cc739)

I would be happy to see the exact same feature in Label page

![Image](https://github.com/user-attachments/assets/8971b8dd-b757-4655-b22e-7ed3f64468d0)

And also when opening a "parent item"

![Image](https://github.com/user-attachments/assets/bec93ff0-3c68-4441-823d-5b1378fa72e1)

### Contributions

- [x] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by 5zud3n** [comment]
eagerly waiting for this feature

**Comment by katosdev** [comment]
Makes sense, and seems like a bit of an oversight not having it - Will need to conduct some testing for performance impact though.

**Comment by katosdev** [comment]
Closing to be consolidated into epic : #436
